jobs/MeterInstanceJob.js

```diff
@@ -0,0 +1,163 @@
+'use strict';
```

Comment:
@abh1kg Have we standardised using `async` `await`. I remember discussing that for new files we shall start using async await. This will be a good start IMO

Reply:
Agreed- in fact, I feel we should use async-await for all cases. The code becomes much cleaner and easier to skim through

Reply:
Updated to use async/await 👍